edb/pgsql/delta.py

```diff
@@ -3587,7 +3587,7 @@ def apply(self, schema, context):
                 dbops.Update(
                     table=table, record=updaterec, condition=condition))
 
-        self.attach_alter_table(context)
+        # self.attach_alter_table(context)
```

Comment:
?

Reply:
Yeah, this is actually a question to you.  What does `self.attach_alter_table(context)`?  The method isn't defined on this class. Commenting it out makes `ALTER MODULE` function seemingly fine.

Reply:
Right. This was misplaced.  It is only for objects that are defined in terms of tables in Postgres.  Remove the line.